build/stage-build.yml

```diff
@@ -52,6 +52,25 @@
       /p:PackageVersion=$(GitVersion.SemVer)
       /p:ContinuousIntegrationBuild=true
 
+- script: >
+    dotnet test $(Build.SourcesDirectory)/$(SolutionFileName)
+    /p:Configuration=$(ApplicationConfiguration)
+    /p:CollectCoverage=true 
+    /p:CoverletOutputFormat=cobertura
+    /p:CoverletOutput=$(Build.SourcesDirectory)/coverage
+    /p:ExcludeByFile="**/*.g.cs"
+    --logger trx
+    --no-build
+  displayName: 'Run tests'
+  condition: and(succeeded(), eq(variables['ApplicationConfiguration'], 'Release'))
```

Comment:
why we want to run them only on release? Shouldn't we always run them for PRs as well?

Reply:
I'll remove the variable eq since apparently it's release all the time